packages/frontend/src/navigation/Nav.tsx

```diff
@@ -34,7 +35,7 @@ export default function Nav(): JSX.Element {
               <Link to="/">
                 {logoSetting && (
                   <img
-                    src={logoSetting.valueRealized as string}
+                    src={imageFullPath(logoSetting.valueRealized as string)}
```

Comment:
Same note as above with getGenericSettingValueRealized -- I think the previous architecture is preferable (where the image's full url is defined by the server).

Reply:
Fixed